prov/rxm/src/rxm.h

```diff
@@ -289,7 +289,7 @@ struct rxm_ep_wire_proto {
 };
 
 struct rxm_cm_data {
-	struct sockaddr name;
+	struct sockaddr_storage name;
```

Comment:
Won't this be an issue when using Verbs core provider based on the limits for user CM data when using the RDMA CM (56 Bytes for RC QP, even less for the XRC implementation)?

Reply:
Yes, this will break rxm over verbs devices.  Note that the rxm_cm_data structure is hopelessly broken though.  The protocol version is offset into the structure (struct sockaddr + u64 bytes), such that we can't reliably check it if we actually change the structure format without holding that offset fixed. Plus it assumes that struct sockaddr is the same on all systems...  We need a redo here.